tests/ord-service/tests/api_test.go

```diff
@@ -67,15 +77,43 @@ func TestORDService(t *testing.T) {
 		eventsMap2[eventDefinition.Name] = *eventDefinition
 	}
 
+	apisMapInScenario := make(map[string]directorSchema.APIDefinitionInput, 0)
+	for _, apiDefinition := range appInputInScenario.Bundles[0].APIDefinitions {
+		apisMapInScenario[apiDefinition.Name] = *apiDefinition
+	}
+
+	eventsMapInScenario := make(map[string]directorSchema.EventDefinitionInput, 0)
+	for _, eventDefinition := range appInputInScenario.Bundles[0].EventDefinitions {
+		eventsMapInScenario[eventDefinition.Name] = *eventDefinition
+	}
+
+	apisMapNotInScenario := make(map[string]directorSchema.APIDefinitionInput, 0)
+	for _, apiDefinition := range appInputNotInScenario.Bundles[0].APIDefinitions {
+		apisMapNotInScenario[apiDefinition.Name] = *apiDefinition
+	}
+
+	eventsMapNotInScenario := make(map[string]directorSchema.EventDefinitionInput, 0)
+	for _, eventDefinition := range appInputNotInScenario.Bundles[0].EventDefinitions {
+		eventsMapNotInScenario[eventDefinition.Name] = *eventDefinition
+	}
```

Comment:
Those are not needed since this app is not asserted anywhere it is used only to verify that it won't be returned since it is not in a scenario.